Applications/Telemetry/telemetry.py

```diff
@@ -587,16 +595,20 @@ def queryStationsDb(parameters):
     sqlEnd = "GROUP BY SOURCECALLSIGN, SOURCEID ORDER BY EPOCH DESC"
 
     # detect if callsign/nodeid provided, return the last time it was heard
-    # Since a callsign was specified, simply search the entire db for it
-    # and return the last epoch time it was heard.
-    timeTuple = (0, time.time())
+    if parameters["CALLSIGN"] != "%":
```

Comment:
Seems to work fine. APRS is working with multiple stations. 1 Local and 1 Remote RF.